stable/nginx-ingress/values.yaml

```diff
@@ -29,6 +29,11 @@ controller:
   # to keep resolving names inside the k8s network, use ClusterFirstWithHostNet.
   dnsPolicy: ClusterFirst
 
+  # Bare-metal considerations via the host network https://kubernetes.github.io/ingress-nginx/deploy/baremetal/#via-the-host-network
+  # Ingress status was blank because there is no Service exposing the NGINX Ingress controller in a configuration using the host network, the default --publish-service flag used in standard cloud setups does not apply
+  reportNodeInternalIp:
```

Comment:
Let's change the value to be just `reportNodeInternalIp: false` without having enabled as a key.

Reply:
Changed `reportNodeInternalIp` as a Boolean.